lib/Sema/CSDiagnostics.cpp

```diff
@@ -9628,6 +9628,12 @@ bool IncorrectInlineArrayLiteralCount::diagnoseAsError() {
   return true;
 }
 
+bool TooManyDynamicMemberLookupsFailure::diagnoseAsError() {
+  emitDiagnostic(diag::too_many_dynamic_member_lookups, Name)
+      .highlight(getAnchor().getSourceRange());
```

Comment:
Nit: Can use `getSourceRange()` here, it already does this by default.

Reply:
Ah nice